Gemfile

```diff
@@ -12,7 +12,7 @@ group :assets do
 
   # Upload assets to AWS S3
   gem 'asset_sync'
-  gem 'fog', '~>1.20', require: 'fog/aws/storage'
+  gem 'fog-aws', '~> 0.12.0', require: 'fog/aws/storage'
```

Comment:
What is the difference? I put that line because it was suggested by https://github.com/AssetSync/asset_sync#optimized-fog-loading.